### DISCUSSION TITLE
Different styles in Chrome vs Firefox (font-medium)

### DISCUSSION THREAD

**Discussion by AdamowiczPawel** [shadcn-ui/ui#3083] [General]
Hello,
I have a problem with displaying styles between browsers (Firefox vs Chrome),

Website https://ui.shadcn.com/ looks the same in Firefox and Chrome.

When I use components from shadcn (after installation according to the Vite instructions), for example, buttons look different in Firefox (font weight is not taken into account).

The problem occurs with font-medium style. As far as I found, this problem is probably related to the lack of font-weight 500.

I tried adding font to index.css however, this does not solve the problem.
`@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700;800&display=swap");`

I believe this is Tailwind problem but does anyone know how to bypass this difference and why there is no such problem on https://ui.shadcn.com?

![image](https://github.com/shadcn-ui/ui/assets/97554315/254a1120-4656-4b9d-97e4-4ba965f32fdb)

Best regards

**Comment by EdoanR** [comment]
This is happening to me too. I managed to make it look *slightly* more similar by increasing the weight of `font-medium` by 1, setting its value to `501` instead of `500`. It still looks different, but in my opinion, it’s less noticeable.

tailwind.config.js:
```diff
module.exports = {
  theme: {
    extend: {
+      fontWeight: {
+       medium: "501",
+      },
    }
}
```

![image](https://github.com/user-attachments/assets/81065c4e-9515-46db-93ac-ab1602aa6cbf)
